### PR TITLE
Schema Validation for @@ignore and @ignore

### PR DESCRIPTION
**Pull request by do4gr**

### CODE REVIEW

File: libs/datamodel/core/src/transform/attributes/id.rs [hunk 1]
```diff
@@ -11,13 +11,10 @@ impl AttributeValidator<dml::Field> for IdAttributeValidator {
     }
 
     fn validate_and_apply(&self, args: &mut Arguments, obj: &mut dml::Field) -> Result<(), DatamodelError> {
+        //todo move this
```
Comment (do4gr) [inline libs/datamodel/core/src/transform/attributes/id.rs:14]: remove comment in next pr

File: query-engine/query-engine/src/tests/dmmf/unsupported.rs [hunk 2]
```diff
@@ -121,6 +121,8 @@ fn relation_with_unsupported_fk_fields_should_be_filtered_from_input_output_type
 
 #[test]
 #[serial]
+
+//todo does this test even make sense anymore? schemavalidation now always requires the ignore
```
Comment (do4gr) [inline query-engine/query-engine/src/tests/dmmf/unsupported.rs:125]: check with @Weakky

File: libs/datamodel/core/tests/attributes/ignore_positive.rs [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+      @@ignore
+    }
+    
+    model ModelNoId {
+      text String 
+     
+      @@ignore
+    }
+
+    /// this does not work yet
```
Comment (tomhoule) [inline libs/datamodel/core/tests/attributes/ignore_positive.rs:56]: is this comment still relevant? I think it works now

**Review by tomhoule — APPROVED** [review]
fantastic

File: libs/datamodel/core/tests/attributes/ignore_positive.rs (hunk 3 above)
Comment (do4gr) [reply libs/datamodel/core/tests/attributes/ignore_positive.rs:56]: true, I made it work

**Pull request merged**